packages/react-cli/src/commands/generate.ts

```diff
@@ -42,6 +46,23 @@ export class GenerateCommand extends Command {
       generatorArgs.push(`--path=${flags.route}/-components`);
     }
 
-    await runEmberInteractively(generatorArgs.join(' '));
+    const answers: any = await inquirer.prompt([
```

Comment:
I was thinking that anything library specific would be its own command, rather than add on to an existing command.

sorta single-responsibility principle here :-\

Reply:
that makes sense, I'm thinking with that I would go back to the generate material component-name and from there the inquirer would kick in with options